rma_calendar_view/view/mrp_repair_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+    <record id="view_repair_order_calendar_form" model="ir.ui.view">
+        <field name="name">mrp.repair.calendar.form</field>
+        <field name="model">mrp.repair</field>
+        <field name="inherit_id" ref="mrp_repair.view_repair_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='guarantee_limit']" position="after">
+                <field name="date_repair"/>
+            </xpath>
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="user_id"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_repair_order_calendar" model="ir.ui.view">
+        <field name="name">Repair Orders Calendar</field>
+        <field name="model">mrp.repair</field>
+        <field name="arch" type="xml">
+            <calendar string="Repair Orders Calendar" date_start="date_repair" color="user_id">
+                <field name="name"/>
+            </calendar>
+        </field>
+    </record>
+
+    <record id="action_repair_order_calendar" model="ir.actions.act_window">
+        <field name="name">Repair Orders Calendar</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">mrp.repair</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">calendar,tree,form</field>
+        <field name="search_view_id" ref="mrp_repair.view_repair_order_form_filter"/>
+        <field name="help" type="html">
+          <p class="oe_view_nocontent_create">
+              Click to create a reparation order. 
+            </p><p>
+              In a repair order, you can detail the components you remove,
+              add or replace and record the time you spent on the different
+              operations.
+            </p><p>
+              The repair order uses the warranty date on the Serial Number in
+              order to know if whether the repair should be invoiced to the
+              customer or not.
+            </p>
+        </field>
+    </record>
+
+    <menuitem id="mrp_repair.menu_repair_order" action="action_repair_order_calendar" parent="mrp.menu_mrp_manufacturing" groups="mrp.group_mrp_user" name="Repair Orders Calendar" sequence="50"/>
```

Comment:
You shouldn't replace full menuitem, but only the property you want to change. Even better, please replace the field in ir.actions.act_window that specify the available views instead of creating a new one.